File: ProblemSolve/1672.cs
/***************************
 * Richest Customer Wealth *
 ***************************/

public class Solution {
    public int MaximumWealth(int[][] accounts) {
        int ans = 0;
        int curSum = 0;
        
        for(int i=0; i<accounts.Length; ++i){
            foreach(int wealth in accounts[i]){
                curSum += wealth;
            }
            
            ans = Math.Max(ans, curSum);
            curSum = 0;
        }
        
        return ans;
    }
}

File: ProblemSolve/1672.cpp
/***************************
 * Richest Customer Wealth *
 ***************************/

class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int ans = 0, curSum = 0;
        int len = accounts.size();
        
        for(int i=0; i<len; ++i){
            for(int wealth : accounts[i]){
                curSum += wealth;
            }
            
            ans = max(ans, curSum);
            curSum = 0;
        }
        
        return ans;
    }
};